cron/worker/main.go

```diff
@@ -217,6 +217,11 @@ func main() {
 	}()
 
 	checksToRun := checks.AllChecks
+	// Remove checks which require lot of GitHub API token.
+	delete(checksToRun, checks.CheckSAST)
+	delete(checksToRun, checks.CheckCITests)
+	// TOOD: Re-add Contributors check after fixing: #859.
+	delete(checksToRun, checks.CheckContributors)
```

Comment:
Also, can you enable branch protection check below. that does not need to disabled since @asraa code went in.

Reply:
I tested `Branch-Protection` locally on `kubernetes` and it returns `branch protection is enabled on development and all release branches but settings are unknown`. IIUC, this info is what we get through `Code-Review` too. Not sure if its worth enabling `Branch-Protection` yet. @laurentsimon @asraa thoughts?

Reply:
that is expected, it will show score=1, the workaround we have won't give furthur info, but we atleast have a differentiator (1=where something is enabled, 0=not enabled)

Reply:
Done. Enabled.